server/bunch/models.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+from django.db import models
+from users.models import User
+
+
+class Bunch(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    owner = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="owned_bunches"
+    )
+    icon = models.ImageField(upload_to="bunch_icons/", null=True, blank=True)
+    is_private = models.BooleanField(default=False)
+    invite_code = models.CharField(max_length=10, unique=True, null=True, blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Member(models.Model):
+    ROLE_CHOICES = [
+        ("owner", "Owner"),
+        ("admin", "Admin"),
+        ("member", "Member"),
+    ]
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Convert to `models.TextChoices`

```suggestion
class RoleChoices(models.TextChoices):
    OWNER = "owner", "Owner"
    ADMIN = "admin", "Admin"
    MEMBER = "member", "Member"

class Member(models.Model):
    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Same for other CHOICES you have defined